java/S3ParquetSink/README.md

```diff
@@ -60,27 +99,102 @@ Flink will [natively and efficiently serialize and deserialize](https://nightlie
-The Hadoop Dependencies are not capable of capturing the AWS Credentials locally in Intellij, for which you will need to provide your Access Key and Secret Access Key in a core-site.xml file.
 
 Steps to follow:
-* Update core-site.xml file with AWS Credentials that have permissions to consume data from a Kinesis Data Stream and write data into Amazon S3
-* Update the flink-conf.yaml and add the path to the core-site.xml file in `fs.hdfs.hadoopconf`
-* You will need to add the following environmental variable before starting the Flink job, please update with path to your flink-conf.file
-  * FLINK_CONF_DIR= `Path to flink-conf.file`
-
-To start the Flink job in IntelliJ edit the Run/Debug configuration enabling 'Add dependencies with "provided" scope to
-the classpath'.
-
-
+* Update `resources/flink-application-properties-dev.json`. Add Kinesis Data Stream Name , Stream Region and S3 path to write the data to.
+```shell
+## Example
+[
+  {
+    "PropertyGroupId": "FlinkApplicationProperties",
+    "PropertyMap": {
+      "input.stream": "stream-input",
+      "stream.region": "ap-south-1",
+      "s3.path": "s3://aksh-flink-sink/flink/msf/"
+    }
+  }
+]
+```
+* Execute with AWS user that has permissions to consume data from a Kinesis Data Stream and write data into Amazon S3.
+* To start the Flink job in IntelliJ edit the Run/Debug configuration enabling 'Add dependencies with "provided" scope to
+  the classpath'.
+* Following is the screenshot of run configuration
+  ![Run Configuration](images/runConfiguration.png)
+
+
+
+## Running locally through Maven command line
+* Update `resources/flink-application-properties-dev.json`. Add Kinesis Data Stream Name , Stream Region and S3 path to write the data to.
+* Execute with AWS user that has permissions to consume data from a Kinesis Data Stream and write data into Amazon S3.
+* Execute following command from the project home directory -
+
+```
+ mvn clean compile exec:java  -Dexec.classpathScope="compile" \
+ -Dexec.mainClass="com.amazonaws.services.msf.StreamingJob" 
+
+```
+
+## Deploying using CloudFormation to Amazon Managed Service for Apache Flink
+### Pre-requisite
+1. Kinesis stream and S3 bucket. 
+2. Create subnets and security groups for the Flink application. If you are using private subnets , ensure that VPC endpoint for Kinesis and S3 is created.
+3. AWS user credential using which you can create CloudFormation stack from console or CLI.
+
+### Build and deployment
+1. The steps below create stack using `./cloudformation/msf-deploy.yaml`.
+2. The script `deploy.sh` creates the stack using AWS CLI. Ensure that AWS CLI is configured and your user has permissions to create CloudFormation stack.
+3. Alternatively you can deploy from console using `./cloudformation/msf-deploy.yaml` and pass required parameters.
+4. Edit `deploy.sh` to modify  "Region and Network configuration" . Modify following configurations -
+* region= Deployment region
+* SecurityGroup= MSK Security Group.
```

Comment:
No MSK in this example. Please remove

Reply:
Edited this out. Network config not needed anymore.